plugin/phpactor.vim

```diff
@@ -1,8 +1,8 @@
-if exists('g:phpactorLoaded')
+if exists('g:loaded_phpactor')
```

Comment:
Would it make sense to name it `phpactorLoaded`? All the other global variables are prefixed with `phpactor` and are camelcase.

Reply:
> it does not follow the recommendation (see commit message)

For me it's an exception we need to keep, in your vim try to expand `:echo g:loaded_` for completion you should see a bunch of them depending on how many plugins you have.
It's given like that in the vim documentation none the less.